example-deployment/README.md

```diff
@@ -246,6 +246,60 @@ deploy it
 .bin/minikube kubectl -- apply -f k8s/dashboard.yaml
 ```
 
+## [MinIO](https://github.com/minio/minio)
```

Comment:
add minio to the list of apps deployed at the top?

Reply:
Done! Thanks!